codegen-core/src/test/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/BuilderGeneratorTest.kt

```diff
@@ -140,4 +143,30 @@ internal class BuilderGeneratorTest {
         }
         project.compileAndTest()
     }
+
+    @Test
+    fun `don't add serde to error types`() {
```

Comment:
Should this be in StructureGeneratorTest, or is it testing both structures and builders?

Reply:
Yes, it is testing both structures and builders. 

I think I should move the test for structures to another file.

Reply:
Nah, this is fine.